The change to the x86-simd-sort submodule now causes an undefined reference
to _ZN12_GLOBAL__N_112avx512_qsortItEEvPT_l on `import numpy`
Reason is that `avx512_qsort<unsigned short>` is used but defined in a header not included
by simd_qsort_16bit.dispatch.cpp when the "AVX512_SPR" version is built.
Fix by including the header.

Author: Alexander Grund (TU Dresden)

diff --git a/numpy/core/src/npysort/simd_qsort_16bit.dispatch.cpp b/numpy/core/src/npysort/simd_qsort_16bit.dispatch.cpp
index 3f5099758..a75f882ff 100644
--- a/numpy/core/src/npysort/simd_qsort_16bit.dispatch.cpp
+++ b/numpy/core/src/npysort/simd_qsort_16bit.dispatch.cpp
@@ -9,6 +9,7 @@
 
 #if defined(NPY_HAVE_AVX512_SPR) && !defined(_MSC_VER)
     #include "x86-simd-sort/src/avx512fp16-16bit-qsort.hpp"
+    #include "x86-simd-sort/src/avx512-16bit-qsort.hpp"
 #elif defined(NPY_HAVE_AVX512_ICL) && !defined(_MSC_VER)
     #include "x86-simd-sort/src/avx512-16bit-qsort.hpp"
 #endif
